### PR TITLE
Add allergies exercise

### PR DESCRIPTION
**Pull request by pfertyk**

### CODE REVIEW

**Comment by pfertyk** [comment]
@BNAndras could you please take a look? ;)

**Comment by BNAndras** [comment]
Looks good. I’ll need to compare with the canonical data.

File: exercises/practice/allergies/allergies_test.gd
Comment (BNAndras) [inline exercises/practice/allergies/allergies_test.gd:1]: I'd suggest using something to break up the test suite into more digestible chunks. The canonical data groups all the eggs tests together for example. I think the easiest would be to add a comment on a line by itself at the beginning of each section. It'd look something like this:

```
last test from previous section

# comment describing the new section

first test from the new section
```

Comment (pfertyk) [reply exercises/practice/allergies/allergies_test.gd:1]: I just copied these tests from the Python track :) I can group them as you suggested though, no problem :+1:

**Comment by pfertyk** [comment]
@BNAndras thank you for your review, I will update my code soon ;) Could you please check if other PRs also need any changes?

**Comment by pfertyk** [comment]
@BNAndras Sorry for the delay :) I've added comments to split the tests into groups, like in the canonical data. I've also added a prefixe to each test to indicate the group.

**Review by BNAndras — APPROVED** [review]

**Pull request merged**